viewer/viewer-awt/src/main/java/org/icepdf/ri/common/utility/search/BaseSearchComponent.java

```diff
@@ -0,0 +1,23 @@
+package org.icepdf.ri.common.utility.search;
+
+import java.text.MessageFormat;
+
+public interface BaseSearchComponent {
```

Comment:
This is picky but I don't like the *Component for the interface name.  In Swing component is usually reserved for actual components.  This could be confusing when someone tries to add the    Maybe  'BaseSearchModel' ' or something along those lines?